Python/src/acas/ACASXu.lf

```diff
@@ -1,95 +1,90 @@
 /**
- * This program models two aircraft moving in a two-dimensional space
- * for testing an Airborne Collision Avoidance System (ACAS) that is
- * realized using neural networks.
- * 
- * It includes a main reactor that applies a simple test where the
- * intruder aircraft follows a straight trajectory and the "own" aircraft
- * maneuvers to avoid a collision.
- * 
+ * This program models two aircraft moving in a two-dimensional space for testing an Airborne
+ * Collision Avoidance System (ACAS) that is realized using neural networks.
+ *
+ * It includes a main reactor that applies a simple test where the intruder aircraft follows a
+ * straight trajectory and the "own" aircraft maneuvers to avoid a collision.
+ *
  * It is based on the following paper:
- * 
- * Arthur Clavière, Laura Altieri Sambartolomé, Eric Asselin,
- * Christophe Garion, ans Claire Pagetti, "Verification of machine
- * learning based cyber-physical systems: a comparative study,"
- * International Conference on Hybrid Systems: Computation and Control
- * (HSCC), May 2022, Pages 1–16, https://doi.org/10.1145/3501710.3519540
  *
- * The original Python code on which this is based was provided by
- * Arthur Clavière and can be found at:
- * 
- *    https://svn.onera.fr/schedmcore/branches/ACAS_CaseStudy/
+ * Arthur Clavière, Laura Altieri Sambartolomé, Eric Asselin, Christophe Garion, ans Claire Pagetti,
+ * "Verification of machine learning based cyber-physical systems: a comparative study,"
+ * International Conference on Hybrid Systems: Computation and Control (HSCC), May 2022, Pages 1–16,
+ * https://doi.org/10.1145/3501710.3519540
+ *
+ * The original Python code on which this is based was provided by Arthur Clavière and can be found
+ * at:
+ *
+ * https://svn.onera.fr/schedmcore/branches/ACAS_CaseStudy/
  *
  * Since that original code bears an LGPL license, so does this program:
- * 
- * The ML models (in code/src/systems/acasxu/nnets)
- * come from https://github.com/guykatzz/ReluplexCav2017
- * 
+ *
+ * The ML models (in code/src/systems/acasxu/nnets) come from
+ * https://github.com/guykatzz/ReluplexCav2017
+ *
  * # Prerequisites
- * 
- * ```
- *    pip install wheel
- *    pip install pandas
- *    pip install matplotlib
- * ```
- * 
+ *
+ * ``` pip install wheel pip install pandas pip install matplotlib ```
```

Comment:
```suggestion
 * ``` pip install wheel && pip install pandas && pip install matplotlib ```
```

Reply:
This is not the correct fix.  The correct fix is to convert this to HTML, I think. The comment was written assuming it would be formatted with markdown.  I suspect all the comments need to be updated to use HTML rather than MD, otherwise the formatter is going to turn the comments into junk.  I think this should be:
```
<pre>
pip install wheel
pip install pandas
pip install matplotlib
</pre>
```

Reply:
We implement the formatter, so we can choose whether to use HTML or MarkDown. I'm not sure why we would prefer HTML.

Reply:
The LF formatter is not the Java formatter. We do fine with markdown syntax (although we don't have a doc-gen system yet). More importantly, the fix looks like a semantic one (not a formatting one). @petervdonovan just added `&&` to make sure the listed command is valid.

Reply:
Reading the comments below, it appears we don't quite "do fine" with markdown, but that wasn't the main point of my comment, anyway...

Reply:
The file `ACASXu.lf` in main looks like this:
```
 * # Prerequisites
 * 
 * ```
 *    pip install wheel
 *    pip install pandas
 *    pip install matplotlib
 * ```
```
The formatter is messing up the markdown. So we need to either change the formatter to do less or we need to convert all the markdown to HTML.

Reply:
Why would HTML do any better? In Java, we use HTML because that's what JavaDoc allows. This is LF code.

Reply:
I think the Problem is really that the formatter also formats the comments and tries to put them on a single line. I think this isn't a good default. See my comment in AlarmClock.lf for another instance if this problem.

Commonly, code formatters either leave comments completely untouched or they only insert line breaks if a line in a comment is too long, but they do not try to merge subsequent lines. I think we should be less aggressive in formatting comments, too.

Reply:
Yes, I agree that many formatters follow such restrictions. Not all of them do. The formatter that we use for our Java code does not do that.

I consider these to be pretty significant restrictions because if a formatter does not handle comments, then developers can spend a lot of time fussing over the aesthetics of their comments and manually re-breaking lines. Furthermore, we already are able to gracefully handle several cases of MarkDown-style formatting. If this is merely a matter of adding a special case for triple backticks, then I think this should be no problem.